client/my-sites/plugins/plugin-meta/index.jsx

```diff
@@ -342,6 +344,15 @@ export default React.createClass( {
 							<div className="plugin-meta__meta">
 								{ this.renderAuthorUrl() }
 							</div>
+							{ ! isEmpty( actionLinks ) &&
+								<div className="plugin-meta__action-links">
+									{ Object.keys( actionLinks ).map( linkTitle => (
+										<Button compact icon href={ actionLinks[ linkTitle ] } target="_blank">
```

Comment:
I think we should add `rel="noopener noreferrer"` here. See https://github.com/Automattic/wp-calypso/pull/7680.